stable/metallb/templates/config.yaml

```diff
@@ -10,13 +10,5 @@ metadata:
     app: {{ template "metallb.name" . }}
 data:
   config: |
-{{- if .Values.config }}
-{{ toYaml .Values.config | indent 4 }}
-{{- else }}
-    address-pools:
-    - name: default
-      protocol: arp
-      cidr:
-      - {{ .Values.arpAddresses }}
-{{- end }}
+{{ toYaml .Values.configInline | indent 4 }}
```

Comment:
Could you change this to `existingConfigMap` to be more in line with other charts (`existingConfigMap`, `existingClaim`, `existingSecret`, ...)